docs/workload-id/workload-identity-federation-config-app-trust-managed-identity.md

```diff
@@ -101,7 +101,7 @@ In this section, you'll configure a federated identity credential on an existing
 Open a terminal in your preferred IDE and run the following command to create a federated identity credential on your app. Replace the GUID with the Object (principal) ID of the managed identity.
 
 ```CLI
-az ad app federated-credential create --id 00001111-aaaa-2222-bbbb-3333cccc4444 --parameters credential.json
+az ad app federated-credential create --id 11112222-bbbb-3333-cccc-4444dddd5555 --parameters credential.json
```

Comment:
This should not be changed, this sample GUID is mentioned again in L113. Please revert.
```suggestion
az ad app federated-credential create --id 00001111-aaaa-2222-bbbb-3333cccc4444 --parameters credential.json
```

Reply:
That is the reason I changed it, I believe the GUID on line 13 should be different. It's my understanding that the Object ID of the managed identity is different than the application ID which becomes confusing when this doc shows the same sample guid for both.